.github/workflows/pack.yml

```diff
@@ -0,0 +1,45 @@
+name: Pack extensions
+
+on:
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [21.x]
```

Comment:
What's the reason for using a matrix strategy if there's only a single node version used?